optuna/_imports.py

```diff
@@ -56,16 +58,24 @@ def __exit__(
         """
         if isinstance(exc_value, (ImportError, SyntaxError)):
             if isinstance(exc_value, ImportError):
+                # TODO(nabenabe0928): Replace the doc with the stable after the v4.0 release.
+                integration_installation_doc_url = (
+                    "https://optuna-integration.readthedocs.io/en/latest/installation.html"
+                )
                 message = (
-                    "Tried to import '{}' but failed. Please make sure that the package is "
-                    "installed correctly to use this feature. Actual error: {}."
-                ).format(exc_value.name, exc_value)
+                    f"\nTried to import '{exc_value.name}' but failed."
+                    "\nPlease make sure that the package is installed correctly."
+                    "\nIf the error relates to optuna-integration, please refer to:"
+                    f"\n\t{integration_installation_doc_url}"
+                    f"\nActual error: {exc_value}."
```

Comment:
I think this is confusing for users who don't use integration modules. I would like to just close this PR.